### DISCUSSION TITLE
Edit PATH in shell and translate bash commands

### DISCUSSION THREAD

**Discussion by theRoboxx** [leo-arch/clifm#186] [Q&A] [answered]
I installed the npm package für Vue.js and would like to add it to my PATH but unfortunate it doesn't work.

bash version: `export PATH=~/.npm-global/bin:$PATH`
my clifm version: `export PATH=~/.npm-global/bin:$(echo "$PATH")`

What am I missing here?

Additionally I just ran into the problem that clifm doesn't translate the `~` symbol to the home directory `/home/roboxx` and therefore the PATH couldn't be set correctly for the following example:

`export PATH=~/.npm-global/bin:/home/roboxx/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl`

**Comment by leo-arch** [comment, marked as answer]
Hi @theRoboxx. All the three ways of setting PATH via **clifm**'s built-in `export` function (exactly as you mentioned them) should work now. Please give a it a try and let me know if it works for you too.

**Reply by theRoboxx** [reply]
Now it works, thanks a lot! :+1:
